src/content/docs/additional/cloud.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Vencloud
+description: Vencords own API
+---
+
+Vencloud is Vencord's API for cloud settings sync!
+
+## Self Hosting
+
+:::caution[Warning]
+Your instance has to be HTTPS capable due to [mixed content restrictions](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content) in web browsers.
+:::
+
+We provide a Docker build, so you don't need anything installed besides Docker!
```

Comment:
Should have link to installing docker compose: https://docs.docker.com/compose/install/